radar_valuelist.go

```diff
@@ -0,0 +1,55 @@
+package stripe
+
+// ValueListItemType is the possible values for a type of value list items.
+type ValueListItemType string
+
+// List of values that ValueListItemType can take.
+const (
+	ValueListItemTypeCardBin             ValueListItemType = "card_bin"
+	ValueListItemTypeCardFingerprint     ValueListItemType = "card_fingerprint"
+	ValueListItemTypeCountry             ValueListItemType = "country"
+	ValueListItemTypeEmail               ValueListItemType = "email"
+	ValueListItemTypeIPAddress           ValueListItemType = "ip_address"
+	ValueListItemTypeString              ValueListItemType = "string"
+	ValueListItemTypeCaseSensitiveString ValueListItemType = "case_sensitive_string"
+)
+
+// ValueListParams is the set of parameters that can be used when creating a value list.
+type ValueListParams struct {
```

Comment:
And likewise, I think for structs in these shared namespaces we're prefixing with the project like `RadarValueListParams`. See what we have in `issuing_cardholder.go` for example.